src/nnet3/nnet-normalize-component.h

```diff
@@ -224,8 +224,8 @@ class BatchNormComponent: public Component {
   const CuVector<BaseFloat> &Offset() const { return offset_; }
   const CuVector<BaseFloat> &Scale() const { return scale_; }
 
-  CuVector<double> &StatsSum() { return stats_sum_; }
-  CuVector<double> &StatsSumsq() { return stats_sumsq_; }
+  CuVector<BaseFloat> Mean();
```

Comment:
CuVector<BaseFloat> Mean()  *******const********;

Reply:
Batchnorm is acutally a affine transform in the inference mode.

Why don't you use `offset` and `scale`?

Reply:
In pytorch's forward calculation, Only we need is put the mean and var to a dict, and use `model.load_state_dict(dst_state_dict)` to load this two variable to pytorch's model, then pytorch can do offset and scale by itself.